game-core/src/main/java/games/strategy/engine/framework/startup/mc/ServerModel.java

```diff
@@ -273,7 +271,6 @@ public void cancel() {
     if (messengers != null) {
       chatController.deactivate();
       messengers.shutDown();
-      messengers.removeErrorListener(this);
```

Comment:
Can't quite follow this change here. Are the messengers guaranteed to never be instances of IClientMessenger?

Reply:
In short, yes, so it's always a no-op. This kind of update I think is the main reason for this kind of refactor.

Line 395: `      messengers = new Messengers(serverMessenger);`, the one place where Messengers is initialized. 

`serverMessenger` is defined line 83:  ` private IServerMessenger serverMessenger;`, and the only implementations are no-op.